src/osemgrep/cli/CLI.ml

```diff
@@ -166,17 +170,15 @@ let dispatch_subcommand argv =
         subcmd_argv0 :: subcmd_args |> Array.of_list
       in
       let experimental = Array.mem "--experimental" argv in
+      (* basic metrics on what was the command *)
+      Metrics_.add_feature "subcommand" subcmd;
+      Metrics_.add_user_agent_tag (spf "command/%s" subcmd);
+      subcmd_argv |> Array.to_list
+      |> exclude (fun x -> not (Base.String.is_prefix ~prefix:"-" x))
```

Comment:
thought we might want to use the `uniq_by` operator to condense multiple usages of the `--config` flag but not picky here.

Reply:
actually I don't think we should support this multiple --config xx, --config yy; I'm not sure it's very UNIXy. I think people should do --config=xx,yy instead.